pkgs/development/libraries/libmemcached/libmemcached-yosemite-ntohll.patch

```diff
@@ -0,0 +1,236 @@
+Originally from Homebrew:
+https://github.com/Homebrew/homebrew/blob/bfd4a0a4626b61c2511fdf573bcbbc6bbe86340e/Library/Formula/libmemcached.rb#L42
```

Comment:
Thank you for the suggestion. I haven't thought that patch will process the ruby code preceding diff as a description :) I've also added links to upstream issues.